pkg/network/node/ovscontroller.go

```diff
@@ -53,18 +54,25 @@ func (oc *ovsController) getVersionNote() string {
 }
 
 func (oc *ovsController) AlreadySetUp(vxlanPort uint32) bool {
-	flows, err := oc.ovs.DumpFlows("table=%d", ruleVersionTable)
-	if err != nil || len(flows) != 1 {
+	ruleVersionFlows, err := oc.ovs.DumpFlows("table=%d", ruleVersionTable)
+	if err != nil || len(ruleVersionFlows) != 1 {
+		return false
+	}
+	// Have a look in the table filled in by setupPodFlows, as to make
```

Comment:
So this is a pre-existing weirdness, but the function calls `DumpFlows("table=%d", ruleVersionTable)` at the top, and then parses it at the bottom, after doing other checks in between for some reason? The two halves of the rule table check should be put back together.